src/topology/algebra/open_subgroup.lean

```diff
@@ -55,44 +55,37 @@ variables {G : Type*} [group G] [topological_space G]
 variables {U V : open_subgroup G} {g : G}
 
 @[to_additive]
-instance has_coe_set : has_coe_t (open_subgroup G) (set G) := ⟨λ U, U.1⟩
+instance has_coe_subgroup : has_coe_t (open_subgroup G) (subgroup G) := ⟨to_subgroup⟩
```

Comment:
I'm not convinced this is a good idea; I don't think these coercions exist for other subobjects.

Reply:
I tried to do as little modifications to API as possible. This coercion was there. Should I try to delete it?

Reply:
Without making it a `coe`, we can't use `norm_cast` on `to_subgroup`. IMHO, we should add a generic `[set_like S G] [subgroup_class S G] : has_coe_t S (subgroup G)` but this can wait.

Reply:
Ah sorry, github produced a confusing diff. I agree we should just leave this.